### PR TITLE
Onboarding-login - first deconstruction

### PR DESCRIPTION
**Pull request by Fodilahy-mena**
Hello team,

I deconstructed Onboarding-login task into two separated parts. The first part is about refactoring some components and displaying the necessary components on Login page which is now ready to review. The second part will be about login functionality that doesn't include in this PR.

Branch link: https://github.com/onja-org/vamalaka-haukea/tree/onboarding-login
Ticket link: https://app.clickup.com/t/41k8af

Thanks in advance!
Jerome

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -46,7 +45,7 @@ const ButtonStyled = styled.button<ButtonProps>`
   color: ${(props) => (props.isPrimary ? '#FFFFFF' : '#041d42')};
   background-color: ${(props) => (props.isPrimary ? '#041d42' : '#FFFFFF')};
   padding: 9px 14px 8px;
-  border: ${(props) => (props.isPrimary ? 'none' : '2px solid #041d42')};
+  border: 2px solid #041D42};
```
Comment (starjardin) [inline src/components/Button/Button.tsx:48]: Why did you change the border?

File: src/components/CommonPageContainer/CommonPageContainer.tsx [hunk 2]
```diff
@@ -90,7 +90,7 @@ const PageContainer = styled.div`
 	}
 `;
 
-const Container = styled.div`
+const Container = styled.main`
```
Comment (starjardin) [inline src/components/CommonPageContainer/CommonPageContainer.tsx:93]: Good change here!

File: src/components/HeaderNavLink/HeaderNavLink.tsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ export const HeaderNavLink: FC<ItemType> = ({ text, imgSrc, alt }) => (
 		<Item>
 			<Link to={`/${text}`} data-testid={text}>
 				<img src={imgSrc} alt={alt} />
-				<span>{text}</span>
+				<span>{alt}</span>
```
Comment (starjardin) [inline src/components/HeaderNavLink/HeaderNavLink.tsx:60]: Why alt, alt can be a long description text.

**Review by starjardin — APPROVED** [review]

**Review by andrzej-onja — CHANGES_REQUESTED** [review]

File: src/components/Button/Button.tsx (hunk 1 above)
Comment (Fodilahy-mena) [reply src/components/Button/Button.tsx:48]: Thanks, I didn't even mean to change it.

File: src/components/HeaderNavLink/HeaderNavLink.tsx (hunk 3 above)
Comment (Fodilahy-mena) [reply src/components/HeaderNavLink/HeaderNavLink.tsx:60]: True!

File: src/Pages/SignIn.tsx [hunk 4]
```diff
@@ -1,16 +1,92 @@
-import { Link } from "react-router-dom"
-import styled from "styled-components"
+import OnboardingPage from './OnboardingPage';
```
[the remark this answers is not in the document]
Comment (Fodilahy-mena) [reply src/Pages/SignIn.tsx:1]: Is the `OnboardingPage` confusing? It's a container component that receives `children` as a prop. It's for DRY code implementation because there is a similarity with SignIn and SignUp pages. Maybe the component name that creates the confusion? Or I don't understand your comment.

File: src/Pages/SignIn.tsx [hunk 5]
```diff
@@ -1,16 +1,94 @@
...
+  onClickLoginOnGoogle: () => void
+  onClickLogin: () => void
+}
+
+const SignIn: React.FC<SigninProps> = ({
+  // loginAccount,
+  // errorMessage,
+  // isChecked,
+  // onChangeEmail,
+  // onChangePassword,
```
Comment (andrzej-onja) [inline src/Pages/SignIn.tsx:42]: deos it complie at all ?
props type still define this

File: src/components/InputName/InputName.tsx [hunk 6]
```diff
@@ -3,9 +3,72 @@ import React, { useState } from 'react'
...
+          value={inputValue}
+          placeholder={placeholder}
+          type={showPassword ? 'text' : inputType}
+          name={inputName}
+          onChange={(e: any) => inputValueHandler(e.target.value)}
+          errorMessage={!isValid}
+          autoComplete='off'
+          required
+        />
+        {inputName === 'password' ? (
```
Comment (andrzej-onja) [inline src/components/InputName/InputName.tsx:54]: shouldnt it be `type ==='password'` ?

**Review by andrzej-onja — APPROVED** [review]

File: src/components/InputName/InputName.tsx (hunk 6 above)
Comment (Fodilahy-mena) [reply src/components/InputName/InputName.tsx:54]: Yes, `type ==='password'` is better. I will change that in my next task. Thanks!
